hardware.json

```diff
@@ -0,0 +1,54 @@
+{
+    "systemservice": {
+        "systemd": {
+            "sudo" : false,
+            "timeoutMs" : 30000,
```

Comment:
Could this be just `timeout`? And we document it's ms? Would be a bit cleaner, I think.

Reply:
Good question, this always leads to confusion. We just have to agree to a standard, stick to it, and if it's not self explaining also document it well.
My idea was to include ms in the name to make it absolutely clear. But it kinda looks awkward if you just deal with the configuration file.
We can change it to `timeout`. Too bad json doesn't allow comment tags...